File: app/src/main/java/com/pepcox/richtar/richtarjakub/data/Beer.java
package com.pepcox.richtar.richtarjakub.data;

public class Beer {

    private String title;
    private String imageUrl;
    private String subtitle;

    public Beer(final String title, final String imageUrl, final String subtitle) {
        this.title = title;
        this.imageUrl = imageUrl;
        this.subtitle = subtitle;
    }

    public String getTitle() {
        return title;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public String getSubtitle() {
        return subtitle;
    }
}

File: app/src/main/java/com/pepcox/richtar/richtarjakub/data/Beer.kt
package com.pepcox.richtar.richtarjakub.data

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import java.io.Serializable

@Entity(tableName = "favorite_beers")
class Beer(
        @PrimaryKey(autoGenerate = true)
        val id: Int,
        @ColumnInfo
        val name: String,
        @ColumnInfo
        val image: String,
        @ColumnInfo
        val detail: String) : Serializable
